WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.kt

```diff
@@ -177,7 +181,6 @@ class PostsListActivity : LocaleAwareActivity(),
 
         quickStartEvent = savedInstanceState?.getParcelable(QuickStartEvent.KEY)
     }
-
```

Comment:
This is only a np, but maybe this line was removed inadvertently?

Reply:
oh you're right! added it back in 9602501bba 🙏